src/zproto_server_c.gsl

```diff
@@ -983,8 +983,18 @@ s_server_handle_protocol (zloop_t *loop, zsock_t *reader, void *argument)
         zmsg_t* recvmsg = zmsg_recv(self->router);
         if(!recvmsg)
             return -1;      //  Interrupted; exit zloop
-        if($(proto)_recv(self->message,recvmsg))
-            return -1; //Interrupted; malformed
+        int rc = $(proto)_recv (self->message, recvmsg);
+        if(rc == -1)
+            return -1;      //  Interrupted; exit zloop
+        if(rc == -2) {
+.if count (class.event, name = "malformed")
```

Comment:
@jdiez17 this behavior is not documented. Could you please explain it in the README. Thanks!